### PR TITLE
Add Integration test with SDK Regression

### PR DESCRIPTION
**Pull request by samarsault**
Support trigger workflows via dispatch from CLI https://github.com/percy/cli/pull/1201 for regression

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "private": true,
+    "scripts": {
+      "test": "percy exec --testing -- bundle exec rspec"
+    },
+    "devDependencies": {
+      "@percy/cli": "^1.16.0"
+    }
+  }
+
```
Comment (itsjwala) [inline package.json:10]: EOF

File: spec/lib/percy/percy_spec.rb [hunk 2]
```diff
@@ -106,3 +106,31 @@
...
+      snap = requests[2]['body']
+      expect(snap['name']).to eq('Name')
+      expect(snap['url']).to eq('http://127.0.0.1:3003/index.html')
+      expect(snap['client_info']).to include('percy-selenium-ruby')
+      expect(snap['environment_info']).to include('selenium')
+      expect(snap['widths']).to eq([375])
+    end
+  end
+
+end
```
Comment (itsjwala) [inline spec/lib/percy/percy_spec.rb:136]: EOF

**Review by itsjwala — APPROVED** [review]

File: spec/lib/percy/percy_spec.rb [hunk 3]
```diff
@@ -106,3 +107,31 @@
...
+      expect(healthcheck['url']).to eq('/percy/healthcheck')
+
+      snap = requests[2]['body']
+      expect(snap['name']).to eq('Name')
+      expect(snap['url']).to eq('http://127.0.0.1:3003/index.html')
+      expect(snap['client_info']).to include('percy-selenium-ruby')
+      expect(snap['environment_info']).to include('selenium')
+      expect(snap['widths']).to eq([375])
+    end
+  end
```
Comment (itsjwala) [inline spec/lib/percy/percy_spec.rb:135]: * you can have nested describe as well, no need to patch rubocop imo.

Comment (samarsault) [reply spec/lib/percy/percy_spec.rb:135]: The before and after steps in the above block mock the percy server which causes this test to fail. That's why its separate

Comment (itsjwala) [reply spec/lib/percy/percy_spec.rb:135]: ohh, I think we can override before(:each) and after(:each) can you try that?

Comment (samarsault) [reply spec/lib/percy/percy_spec.rb:135]: Already tried, it executes both

**Review by itsjwala — APPROVED** [review]

**Pull request merged**